src/components/ContactUs/Form.js

```diff
@@ -11,7 +11,7 @@ function Form() {
     emailjs
```

Comment:
This emailing function seems nice, so it can live in another file that we will set up for our helper functions. Can you try to move the function to a file called utils.js inside a utils folder. That way we can import it and use it so the code will look much cleaner.